hooks/validator/README.md

```diff
@@ -0,0 +1,89 @@
+# Validator Hook
+
+![Experimental](https://img.shields.io/badge/experimental-breaking%20changes%20allowed-yellow)
+![Alpha](https://img.shields.io/badge/alpha-release-red)
```

Comment:
It's up to you but these badges can likely be removed.

Reply:
Done